Makefile

```diff
@@ -28,7 +28,7 @@ build:
 	@echo "------------------------------------"
      
 	helm init --client-only
```

Comment:
We can remove this now, helm 3 does not need this anymore

Reply:
For this we will have to update helm3 in the CI pipeline  as well. Can we let this merge go through and then then update helm operator in CI pipeline in another PR?